components/landing/TheQuickStart/Algorithms.vue

```diff
@@ -0,0 +1,169 @@
+  <section class="algorithms">
+    <div class="algorithms__container">
+      <h3>Get the SDK</h3>
+      <div class="algorithms__section">
+        <code class="algorithms__code-cell">
+          <pre class="algorithms__code-cell__line">pip install qiskit</pre>
+          <pre class="algorithms__code-cell__line">pip install qiskit-ibm-provider</pre>
+        </code>
+        <!--
+          @click="[copyToClipboard($event), $trackClickEvent(`Copy ${title} install code`, 'providers')]"
+        -->
+        <cv-button
+          title="Copy"
+          size="sm"
+          class="algorithms__copy-button"
+          @click="copyToClipboard($event)"
+        >
+          Copy
+        </cv-button>
+      </div>
+      <h3>Test some algorithms</h3>
+      <cv-tabs container>
+        <cv-tab v-for="algorithm in algorithmsContent" :key="algorithm.title" :label="algorithm.title">
+          <div class="algorithms__section">
+            <code class="algorithms__code-cell">
+              <template v-for="(line, index) in algorithm.codeSample">
+                <!-- eslint-disable vue/no-v-html -->
+                <pre v-if="line.startsWith('#')" :key="line" class="algorithms__code-cell__comment" v-html="line" />
+                <pre v-else :key="index" class="algorithms__code-cell__line" v-html="line" />
+                <!-- eslint-enable -->
+              </template>
+            </code>
+            <cv-button
+              title="Copy"
+              size="sm"
+              class="algorithms__copy-button"
+              @click="copyToClipboard($event)"
+            >
+              Copy
+            </cv-button>
+          </div>
+        </cv-tab>
+      </cv-tabs>
+    </div>
+  </section>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import { Component } from 'vue-property-decorator'
+import { AlgorithmsContent } from '~/constants/providersContent'
+
+@Component
+export default class Algorithms extends Vue {
+  copyCodeCta = 'Copy'
+  algorithmsContent = AlgorithmsContent
+
+  copyToClipboard (e: { target: any }): void {
+    const buttonElement = e.target
+    const codeBlock = buttonElement.parentNode
+    const codeSnippet = codeBlock.querySelector('.algorithms__code-cell')
+    if (codeSnippet !== null) {
+      navigator.clipboard.writeText(codeSnippet.innerText)
+      buttonElement.innerText = 'Copied!'
+      setTimeout(() => {
+        buttonElement.blur()
+        buttonElement.innerText = this.copyCodeCta
+      }, 3500)
+    }
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+$cta-max-width: 4rem;
+
+.algorithms {
+  padding: $spacing-05;
+
+  &__container {
+    background-color: $background-color-lighter;
+    padding: $spacing-05;
+  }
+
+  &__section {
+    position: relative;
+    margin-bottom: $spacing-07;
+  }
+
+  &__code-cell {
+    display: block;
+    background-color: $background-color-white;
+    color: $text-color;
+    padding: $spacing-05 $spacing-12 $spacing-05 $spacing-05;
+
+    &__comment {
```

Comment:
Correct me if I'm wrong, but I think `comment` is a modifier, so it should be `_comment` instead of `__comment`

Reply:
You're right! Fixed in https://github.com/Qiskit/qiskit.org/pull/3020/commits/d60f48b9a31ffecbe3e17f577ebfa01892204e93